capsules/core/src/virtualizers/virtual_i2c.rs

```diff
@@ -17,10 +17,12 @@ use kernel::utilities::cells::{OptionalCell, TakeCell};
 pub struct MuxI2C<'a, I: i2c::I2CMaster<'a>, S: i2c::SMBusMaster<'a> = NoSMBus> {
     i2c: &'a I,
     smbus: Option<&'a S>,
-    i2c_devices: List<'a, I2CDevice<'a, I, S>>,
+    i2c_devices: List<'a, I2CDevice<'a, I, false, S>>,
```

Comment:
Rather than adding a bool, I think we should just have a separate object in the `MuxI2C` struct. The "rest of the bus" isn't really a device.

Reply:
We do add a `i2c_bus` object to the `MuxI2C` object.

The bool is just to reuse the existing `I2CDevice` logic instead of copying everything

Reply:
I think its ok to copy to get the new functionality and have it be clear and maintainable.